plan/plan_test.go

```diff
@@ -103,6 +104,14 @@ func (suite *PlanTestSuite) SetupTest() {
 			endpoint.ResourceLabelKey: "ingress/default/foo-5",
 		},
 	}
+	suite.fooAAAA = &endpoint.Endpoint{
+		DNSName:    "foo",
+		Targets:    endpoint.Targets{"2001:DB8::1"},
+		RecordType: "AAAA",
+		Labels: map[string]string{
+			endpoint.ResourceLabelKey: "ingress/default/foo-AAAA",
+		},
+	}
```

Comment:
Where is this used?

Reply:
Currently nowhere, but the idea is to use it for a actual test.

Reply:
I believe there should be a test for a `DNSName` that has an IPv6 record and another test for a `DNSName` that has both an IPv4 and IPv6 address.

Reply:
I believe I have now done those.